airflow/api_connexion/openapi/v1.yaml

```diff
@@ -1679,7 +1704,6 @@ components:
               name:
                 type: string
             nullable: true
-          readOnly: true
```

Comment:
Which this change?

Reply:
So we can assign the user to roles on creation.

Reply:
oh well, my bad I confused it for RoleSchema 🤦